flows/routers/cases/tests.go

```diff
@@ -849,7 +851,7 @@ type decimalTest func(value decimal.Decimal, test1 decimal.Decimal, test2 decima
 
 func testNumber(env utils.Environment, str types.XText, testNum1 types.XNumber, testNum2 types.XNumber, testFunc decimalTest) types.XValue {
 	// create a number finding regex based on current environment
-	pattern := regexp.MustCompile(fmt.Sprintf(`[-+]?[\pNlO\%s]+(\%s[\pNlO]+)?`, env.NumberFormat().DigitGroupingSymbol, env.NumberFormat().DecimalSymbol))
+	pattern := regexp.MustCompile(fmt.Sprintf(`[-+]?([\pN\%[1]s]+(\%[2]s[\pN]+)?|(\W|^)\%[2]s[\pN]+)`, env.NumberFormat().DigitGroupingSymbol, env.NumberFormat().DecimalSymbol))
```

Comment:
note removal of `lO` (lowercase L, uppercase O) here - we ripped out doing substitutions of these but I evidentally forgot to remove them from this regex, meaning `1OO` would match here but then fail in `ParseDecimalFuzzy`. Now we match that as just `1`